concepts/0017-attachments/README.md

```diff
@@ -154,6 +151,31 @@ following decorator:
 
 [![appended attachments](appended.png)](crime-scene.json)
 
+### Signing attachments
+
+In some cases it may be desirable to sign an attachment in addition to or instead of signing the message as a whole. Consider a home-buying protocol; the home inspection needs to be signed even when it is removed from a messaging flow.
+
+Embedded and appended attachments support this by replacing the `data.base64` field with a `sig` field that uses [JWS format](https://openid.net/specs/draft-jones-json-web-signature-04.html), where the payload of the JWS is the data that would otherwise have been base64-encoded:
```

Comment:
prefer link to the non-draft IETF RFC: https://tools.ietf.org/html/rfc7515

Reply:
Updated.